src/Core/Graphics/LevelSprite.cpp

```diff
@@ -122,6 +122,11 @@ namespace obe::Graphics
         m_layerChanged = true;
     }
 
+    void LevelSprite::setAntiAliasing(bool antiAliasing)
+    {
+        m_antiAliasing = antiAliasing;
```

Comment:
It seems the Texture won't be updated even if you change the antialiasing setting